prompts/unit_testing.yaml

```diff
@@ -0,0 +1,41 @@
+- name: "Create Single Module Unit Tests"
+  description: "Prompt for Devin to create unit tests for a specific Python module, following specific guidelines and workflow."
+  target_ai: "Devin"
+  details:
+    module_url: "https://github.com/venture-alaska-tango/eo-push-risk/blob/f5c7c612b39168c891c447b259ba148db6581867/eo_push_risk/feature_engineering/features/text_features.py"
```

Comment:
@mbarrosouplabs  let's make and use a tag URL if we still decide to stick with github (vs mlflow as a storage)